README.md

```diff
@@ -98,44 +100,29 @@ Most of the environment variables are set to default values. This is the short l
 of the most common ones with non default values. For more info take a look at the file
 [docker-compose-base.yml](docker-compose-base.yml).
 
-#### Aether Kernel
-
-- `DB_NAME`: `aether` Database name.
-- `WEB_SERVER_PORT`: `8100` Web server port.
-- `ADMIN_TOKEN`: `kernel_admin_user_auth_token`
-  to connect to it from other modules. It's used within the start up scripts.
-- `CSV_SEPARATOR`: `,` fields separator in the CSV export format.
-
-#### Aether ODK Module
-
-- `DB_NAME`: `odk` Database name.
-- `WEB_SERVER_PORT`: `8102` Web server port.
-- `ADMIN_TOKEN`: `odk_admin_user_auth_token`
-  to connect to it from other modules. It's used within the start up scripts.
-- `AETHER_KERNEL_TOKEN`: `kernel_any_user_auth_token` Token to connect to kernel server.
-- `AETHER_KERNEL_URL`: `http://kernel:8100` Aether Kernel Server url.
-- `AETHER_KERNEL_URL_TEST`: `http://kernel-test:9100` Aether Kernel Testing Server url.
-
-#### Aether UI
+#### Generic
+
+- `ADMIN_USERNAME`: `admin` The setup scripts create an initial admin user for the app.
+- `ADMIN_PASSWORD`: `secresecret`.
+- `ADMIN_TOKEN`: `admin_user_auth_token` Used to connect from other modules.
+- `APP_LINK`: `http://aether.ehealthafrica.org`. The link that appears in the DRF web pages.
+- `APP_NAME`: `aether`. The app name displayed in the web pages.
+- `APP_URL`, `/`. The app url in the server.
+  If host is `http://my-server.org` and the app url is `/my-dev-instance`,
+  the app enpoints will be accessible at `http://my-server.org/my-dev-instance/...`.
+  The local behavior is: `http://my-module.aether.local/`.
```

Comment:
But - just to be clear - if this is set as in the example, it will also (only?) be accessible at `http://aether.local/my-module/`, right?

Reply:
I added a longer paragraph with a better explanation :nerd_face: